tests/functional/disaster-recovery/regional-dr/test_dryrun_failover.py

```
@@ -0,0 +1,688 @@
+        for info in workload_info:
+            logger.info(
+                f"[{info['drpc_resource_name']}] Waiting for phase "
+                f"'{constants.STATUS_FAILEDOVER}'"
+            )
+            info["drpc_obj"].wait_for_phase(
+                constants.STATUS_FAILEDOVER,
+                timeout=360,
+            )
+
+        # Recover cluster_a
+        logger.info(f"Recovering cluster_a '{cluster_a}' nodes after promote")
```

Ruff (0.15.21)

[warning] Logging statement uses f-string

(G004)